src/context/runTest.js

```diff
@@ -4,8 +4,10 @@ const headers = { 'Content-Type': 'application/json' }
 
 export default function runTest(snapshot, update) {
   const {
-    host = 'localhost',
-    port = 3000,
+    serverInfo: {
+      host = 'localhost',
```

Comment:
But I've a question about the removal. In `runTest.spec` we execute the module outside webpack and passing a deeply structured object manually to `process.env.SNAPGUIDIST ` doesn't works.

This:
```javascript
process.env.SNAPGUIDIST = { serverInfo: { host: 'localhost', port: 3000 } }
``` 

returns `'[object Object]'`

This pass the contents but the `runTest` module receive it as a string and not an object.
```javascript
process.env.SNAPGUIDIST = JSON.stringify({ serverInfo: { host: 'localhost', port: 3000 } })
``` 

Do you have any idea on how to fix that situation?